.github/workflows/maintenance.yaml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches: [ master ]
+  schedule:
+    - cron: '*/5 * * * *'
+  workflow_dispatch: # Enables on-demand/manual triggering
+jobs:
+  job:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - run: |
+          go get github.com/aws/aws-sdk-go
+          go mod tidy
+          go generate ./...
+          cd tools/provider-schema
+          terraform init -upgrade
+          terraform providers schema -json > schema.json
+      - uses: peter-evans/create-pull-request@v3
```

Comment:
Answering my own question here, this will continuously update the PR

https://github.com/peter-evans/create-pull-request#action-behaviour